vmdb/tools/fix_auth/rails_mock.rb

```diff
@@ -0,0 +1,11 @@
+unless defined?(Rails)
+  module Rails
+    def self.rails_root
+      @rails_root ||= ENV["RAILS_ROOT"]
+    end
+
+    def self.env
+      @env ||= ActiveSupport::StringInquirer.new("production")
```

Comment:
I'm concerned with this.  The whole reason this PR exists is because we're requiring 'activerecord' in a ruby process so `Rails` isn't defined.  This PR implies that `ActiveSupport` needs  to be defined (even if it's already required by ActiveRecord).  I'd prefer https://github.com/ManageIQ/rails/pull/16 as it's less surprising.